libs/vkd3d/device.c

```diff
@@ -3728,6 +3728,12 @@ static HRESULT d3d12_device_get_format_support(struct d3d12_device *device, D3D1
     const struct vkd3d_format *format;
     VkResult vr;
 
+    if (!is_valid_format(data->Format))
```

Comment:
Return this after setting Support1/2 to NONE. Getting some warnings about uninitialized variable.